CODEOWNERS

```diff
@@ -1,7 +1,7 @@
-* @reset @fnichol @christophermaier
+* @reset @fnichol @christophermaier @raskchanky
 .buildkite @christophermaier @fnichol @scotthain
 components/backline @fnichol @baumanj
-components/builder* @chefsalim @raskchanky
+components/builder-api-client @chefsalim @raskchanky
```

Comment:
Yeah, that's fair.  I made this because I was confused as to why it didn't consider me a reviewer for one of the PRs that came through recently that mainly touched `builder-api-client`, but this might not actually be an improvement.  I'll put it back.

Reply:
Weird. I'll to a quick test and make sure that's working correctly.

Reply:
Seems ok, yeah? https://github.com/habitat-sh/habitat/pull/5526

Reply:
Yeah, I wonder what I saw earlier.  Thanks for testing.